x11-libs/xbae/files/xbae-4.60.4-c23.patch

```diff
@@ -0,0 +1,167 @@
+Fix two remaining problems for C99/C23 compatibility.
+With this, package builds with GCC-15.
+true and false are strings, so replacement is trivial
+and doesn't interact with ABI.
+Casts to Widget are uses everywhere in this program, and
+are correct, as MatrixWidget embeds same core struct from Xt
+as Widget consists of, as first member.
+XtRImmediate indicates that next pointer would be interpreted
+as a value, so integer reinterpreted as a pointer to low mem is
+actually fine, it won't be dereferenced and won't crash whole
+program.
+https://bugs.gentoo.org/944748
+https://bugs.gentoo.org/919195
+--- a/examples/input/input.c
++++ b/examples/input/input.c
```

Comment:
Since this file is just an example, do we really need to patch it?

Reply:
Examples USE flag controls only if examples are installed or not.
building them is part of all-recursive target, and patching the file is easier, for me, than working around recursive targets and default src_compile()